app/src/forms/admin/routes.js

```diff
@@ -18,22 +18,175 @@ routes.use(currentUser);
 //
 // Forms
 //
+
+/**
+ * @openapi
+ * /admin/forms:
+ *  get:
+ *    tags:
+ *      - Admin
+ *    summary: List all forms in the system
+ *    description: This endpoint will fetch all forms under the admin access.
+ *    security:
+ *      - openId: []
+ *    responses:
+ *      '200':
+ *        description: Success
+ *        content:
+ *          application/json:
+ *            schema:
+ *              $ref: '#/components/responses/responseBody/AdminsFormGetEx'
+ *      '403':
+ *        $ref: '#/components/responses/Error/AccessDenied'
+ *      '5XX':
+ *        $ref: '#/components/responses/Error/UnExpected'
+ */
 routes.get('/forms', async (req, res, next) => {
   await controller.listForms(req, res, next);
 });
 
+/**
+ * @openapi
+ * /admin/forms/{formId}:
+ *  get:
+ *    tags:
+ *      - Admin
+ *    summary: Get top level details of a form
+ *    description: This endpoint will fetch the form details(and metadata for versions).
+ *    security:
+ *      - openId: []
+ *    parameters:
+ *      - in: path
+ *        name: formId
+ *        schema:
+ *          type: string
+ *          format: uuid
+ *        description: ID of the form.
+ *        required: true
+ *        example: c6455376-382c-439d-a811-0381a012d696
+ *    responses:
+ *      '200':
+ *        description: Success
+ *        content:
+ *          application/json:
+ *            schema:
+ *              $ref: '#/components/responses/responseBody/AdminReadFormEx'
+ *      '403':
+ *        $ref: '#/components/responses/Error/AccessDenied'
+ *      '5XX':
+ *        $ref: '#/components/responses/Error/UnExpected'
+ *      '404':
+ *        $ref: '#/components/responses/Error/ResourceNotFound'
+ */
 routes.get('/forms/:formId', async (req, res, next) => {
   await controller.readForm(req, res, next);
 });
 
+/**
+ * @openapi
+ * /admin/forms/{formId}/apiKey:
+ *  delete:
+ *    tags:
+ *      - Admin
+ *    summary: Delete Form API Key
+ *    description: This endpoint will delete the form API key.
+ *    security:
+ *      - openId: []
+ *      - bearerAuth: []
+ *    parameters:
+ *      - in: path
+ *        name: formId
+ *        schema:
+ *          type: string
+ *          format: uuid
+ *        description: ID of the form.
+ *        required: true
+ *        example: c6455376-382c-439d-a811-0381a012d696
+ *    responses:
+ *      '200':
+ *        description: Success
```

Comment:
The old docs say that it returns a 204. Were the old docs incorrect? We definitely want to match what's being returned.